src/geo/map/torque-layer.js

```diff
@@ -42,10 +39,12 @@ var TorqueLayer = LayerModelBase.extend({
   },
 
   _onAttributeChanged: function () {
-    var reloadVis = _.any(this.ATTRIBUTES_THAT_TRIGGER_VIS_RELOAD, function (attr) {
+    var reloadVis = _.any(ATTRIBUTES_THAT_TRIGGER_VIS_RELOAD, function (attr) {
       if (this.hasChanged(attr)) {
         if (attr === 'cartocss') {
-          return this.previous('cartocss') && this._torqueCartoCSSPropsChanged();
+          var hasTorquePropsChanged = this._torqueCartoCSSPropsChanged();
+          var usesTurboCarto = this._hasTurboCarto();
```

Comment:
Personal detail... I would remove these intermediate variables since they are only used once and are very similar to invoking the method directly...